x-pack/test/functional/apps/lens/group1/ad_hoc_data_view.ts

```diff
@@ -230,6 +256,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       // adhoc data view should be persisted after refresh
       await browser.refresh();
       await checkDiscoverNavigationResult();
+
+      await browser.closeCurrentWindow();
+      await browser.switchToWindow(daashboardHandle);
```

Comment:
This is a fix for a bug on the functional tests suite.

Sometimes the secondary tab with discover was kept open (didn't understand why...) and the following suites were carried on on that tab, leaving the first one open, and confusing other suite who usually have code like that to switch between two tabs:

```ts
const [currentLensTab, otherTabs] = await browser.getAllWindowHandles();
```

but because the first tab is no longer the current Lens instance, rather a stale dashboard page, then all checks fail.